cmd/libsnap-confine-private/snap.h

```diff
@@ -37,6 +37,13 @@ enum {
 	SC_SNAP_INVALID_INSTANCE_NAME = 3,
 };
 
+/* 40 char snap_name */
```

Comment:
I would state something like:
```
SNAP_NAME_LEN is the maximum length of a snap name, enforced by snapd and the store.
```

Similarly for the other two, as otherwise the comments really say nothing new.